src/clientiface.h

```diff
@@ -431,38 +431,39 @@ class ClientInterface {
 	const std::vector<std::string> &getPlayerNames() const { return m_clients_names; }
 
 	/* send message to client */
-	void send(u16 peer_id, u8 channelnum, NetworkPacket* pkt, bool reliable);
+	void send(SessionId peer_id, u8 channelnum, NetworkPacket* pkt, bool reliable);
 
 	/* send to all clients */
 	void sendToAll(NetworkPacket *pkt);
 	void sendToAllCompat(NetworkPacket *pkt, NetworkPacket *legacypkt, u16 min_proto_ver);
 
 	/* delete a client */
-	void DeleteClient(u16 peer_id);
+	void DeleteClient(SessionId peer_id);
 
 	/* create client */
-	void CreateClient(u16 peer_id);
+	void CreateClient(SessionId peer_id);
 
 	/* get a client by peer_id */
-	RemoteClient* getClientNoEx(u16 peer_id,  ClientState state_min=CS_Active);
+	RemoteClient* getClientNoEx(SessionId peer_id,  ClientState state_min=CS_Active);
 
 	/* get client by peer_id (make sure you have list lock before!*/
-	RemoteClient* lockedGetClientNoEx(u16 peer_id,  ClientState state_min=CS_Active);
+	RemoteClient* lockedGetClientNoEx(SessionId peer_id,  ClientState state_min=CS_Active);
 
 	/* get state of client by id*/
-	ClientState getClientState(u16 peer_id);
+	ClientState getClientState(SessionId peer_id);
 
 	/* set client playername */
-	void setPlayerName(u16 peer_id,std::string name);
+	void setPlayerName(SessionId peer_id,std::string name);
```

Comment:
Same formatting problems here (and above). Could you please correct the lines you have changed?
Superfluous spaces, missing spaces and misplaced pointers.

Reply:
possible